packages/components/entity-list/stories/EntityList.stories.tsx

```diff
@@ -12,30 +13,75 @@ export default {
   },
 } as Meta;
 
+const actions = [
+  <MenuSectionTitle
+    key="title"
+    onClick={() => {
+      console.log('[ <MenuSectionTitle> ] title');
+    }}
+  >
+    Actions
+  </MenuSectionTitle>,
+  <MenuItem
+    key="edit"
+    onClick={() => {
+      console.log('[ <MenuItem> ] Edit');
+    }}
+  >
+    Edit
+  </MenuItem>,
+  <MenuItem
+    key="cancel"
+    onClick={() => {
+      console.log('[ <MenuItem> ] Delete');
+    }}
+  >
+    Delete
+  </MenuItem>,
+];
+
 export const Basic: Story<EntityListProps> = () => (
   <EntityList>
     <EntityList.Item
```

Comment:
I could take or leave these storybook changes.  I found them very useful while developing, mostly the changes are debugging in nature, like updating the title to reflect the eventual type of element that will be rendered `button` or `a`, and adding onClick console messages to ensure that callbacks are firing correctly.

Reply:
i think it's good to have these examples, let's keep them!